src/main/java/org/ow2/proactive/catalogclient/util/ResponseUtils.java

```diff
@@ -40,4 +44,9 @@ public static void checkResponse(Response response) {
             throw new FailedRequestException(response.getStatusCode() + " " + response.getStatusLine());
         }
     }
+
+    private static void sss() throws FileNotFoundException {
+        FileOutputStream out = new FileOutputStream(new File(""));
+        FileOutputStream out1 = new FileOutputStream(new File(""));
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Use try-with-resources or close this "FileOutputStream" in a "finally" clause. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2095)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "out1". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1854)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "out1" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1481)